homeassistant/components/broadlink/sensor.py

```diff
@@ -51,12 +56,12 @@
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the Broadlink device sensors."""
     host = config.get(CONF_HOST)
```

Comment:
Use `dict[key]` for required and optional config keys with a default value.
```suggestion
    host = config[CONF_HOST)]
```